Example/Kommunicate_ExampleUITests/KommunicateCreateConversationAndSendMessagesTests.swift

```diff
@@ -136,6 +136,19 @@ class KommunicateCreateConversationAndSendMessagesTests: XCTestCase {
         return app
     }
     
+    override func tearDown() {
+        let app = XCUIApplication()
+        let inConversationbackButton = app.navigationBars[AppScreen.myChatScreen]
+        waitFor(object: inConversationbackButton) { $0.exists }
+        inConversationbackButton.buttons[InAppButton.ConversationScreen.backButton].tap()
+        let backButton = app.navigationBars[AppScreen.myChatScreen]
+        waitFor(object: backButton) { $0.exists }
+        backButton.buttons[InAppButton.ConversationScreen.back].tap()
+        let logoutButton = app.staticTexts[InAppButton.LaunchScreen.logoutButton]
+        logoutButton.tap()
+        super.tearDown()
```

Comment:
We don't have to log out after each test. We can call log out after one of the tests or create a separate test.

Reply:
Done removed the TearDown function.